docs/src/styles.css

```diff
@@ -1,46 +1,52 @@
-@import './fonts/fonts.css';
+@import './fonts/index.css';
+@import './syntax-highlighting/index.css';
+@import './components/demo/index.css';
+@import './components/CodeBlock.css';
 
-@import 'tailwindcss/base';
-@import 'tailwindcss/components';
-@import 'tailwindcss/utilities';
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
 
 :root {
-  --color-content: #fff;
+  --font-mono: 'Menlo', 'Consolas', monospace;
```

Comment:
Is the default font OK for Linux users?

Reply:
Good point, I'll add a couple more fallbacks